examples/deadletterqueue/movingMessagesToDLQ.ts

```diff
@@ -0,0 +1,192 @@
+import {
+  delay,
+  generateUuid,
+  SendableMessageInfo,
+  OnMessage,
+  OnError,
+  MessagingError,
+  ServiceBusMessage,
+  ReceiveMode,
+  Namespace,
+  QueueClient
+} from "../../lib";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const str = process.env.SERVICEBUS_CONNECTION_STRING || "";
+const queuePath = process.env.QUEUE_NAME || "";
+const numberOfMessages: number = parseInt(process.env.MESSAGE_COUNT || "1");
+const receiveClientTimeout: number = 10000;
```

Comment:
You don't need to specify `: number`

I would also make it clear what unit of time this is using (I assume this is miliseconds)

Reply:
Updated.